metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
I know the spelling isn't great, but then will people be looking for a CIN function? Calling it anything else also inhibits discovery.